kubernetes/windows/setup.ps1

```diff
@@ -42,7 +42,7 @@ Write-Host ('Finished Installing Fluentbit')
 Write-Host ('Installing Telegraf');
 try {
     # For next telegraf update, make sure to update config changes in telegraf.conf, tomlparser-prom-customconfig.rb and tomlparser-osm-config.rb
-    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.24.2_windows_amd64.zip'
+    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.29.4_windows_amd64.zip'
```

Comment:
is this version working? in the past, we had issues with telegraf update  on windows hence we didnt upgrade the version.